tasks/newer.js

```diff
@@ -96,11 +116,13 @@ function createTask(grunt) {
         });
       }
 
-      // configure target with only newer files
-      config.files = newerFiles;
-      delete config.src;
-      delete config.dest;
-      grunt.config.set([taskName, targetName], config);
+      if(options.changeConfig !== false) {
```

Comment:
for this to work correctly we to check if any files have changed. the current result of this code is the same as not prepending newer. 

i'm not a fan of "changeConfig" as the setting. we're looking to stop 'newer' from filtering files. how about 'filter' for the config name.

I think a bit more logic needs to be addressed here.